backend/app/utils.py

```diff
@@ -108,8 +108,18 @@ def appstream2dict(appstream_url=None) -> dict[str, dict]:
                         "".join(description),
                     )
 
-        screenshots = component.find("screenshots")
-        if screenshots is not None:
+        screenshotsComponent = component.find("screenshots")
+
+        if screenshotsComponent is not None:
+            screenshots = [
+                screenshot
+                for screenshot in screenshotsComponent
+                if screenshot.attrib.get("environment")
+                not in (
+                    "windows",
+                    "macos",
+                )
+            ]
```

Comment:
```suggestion
        if screenshotsComponent is not None:
            if not all(
                [
                    screenshot.attrib.get("environment")
                    in (
                        "windows",
                        "macos",
                    )
                    for screenshot in screenshotsComponent
                ]
            ):
                screenshots = [
                    screenshot
                    for screenshot in screenshotsComponent
                    if screenshot.attrib.get("environment")
                    not in (
                        "windows",
                        "macos",
                    )
                ]
            else:
                screenshots = [screenshot for screenshot in screenshotsComponent]
```

Reply:
Should we have a test case if this is important?

Reply:
I don't mind either way with tests.

I had the fallback bevause I don't want the backend to end uo with no screenshots just because someone incorrectly tagged them. We use what we have.